src/components/RecordButton.js

```diff
@@ -6,15 +6,17 @@ import isRecordingButton from '../images/recording-wave.gif';
 export default function Recorder({
   recordStopClick,
   recordStartClick,
-  recordingSrc,
+  devicesState,
+  setDevicesState,
 }) {
   const [isRecording, setIsRecording] = useState(false);
 
   useEffect(() => {
-    if (recordingSrc !== '') {
+    if (devicesState !== '') {
       setIsRecording(false);
+      setDevicesState('');
```

Comment:
Is it the same like in the DrumLoopPlayer.js? Is it needed in both components?

Reply:
Yes, the audio recorder and the drum loop player have to stop each other in certain cases. This state then changes the image of the buttons to the initial value.